### ISSUE TITLE
Docker container --cap-add / --cap-drop support

### ISSUE THREAD

**Issue by rojaro** [hashicorp/terraform#11623]
## Feature Request

When creating new Docker containers using the **docker_container resource**, it is currently not possible to configure specific Linux capabilities i.e. to add the NET_ADMIN capability to a container. 

```
$ docker run --help | grep cap
      --cap-add value               Add Linux capabilities (default [])
      --cap-drop value              Drop Linux capabilities (default [])
```

**Comment by dmportella** [comment]
I would happy to make these changes

**Comment by mengesb** [comment]
OMG i so need this! would help in running the dnsmasq container!!!

**Comment by dmportella** [comment]
@mengesb PR is in and working just needs to wait for comments and merging.

**Comment by mengesb** [comment]
Seems I managed to work around this wrt dnsmasq (I think... i hope) however I'm sure it's still going to be useful

**Comment by stack72** [comment]
Closed via #12045
